content/docs/guides/time-travel-assist.md

```diff
@@ -26,7 +26,7 @@ However, you can see the history of operations related to the creation and delet
 
 ### How long do ephemeral endpoints remain active
 
-The ephemeral endpoints are created according to your configured [default compute size](/docs/manage/projects#reset-the-default-compute-size). An ephemeral compute remains active for as long as you keep running queries against it. After 10 seconds of inactivity, the timeline is deleted and the endpoint is removed.
+The ephemeral endpoints are created according to your configured [default compute size](/docs/manage/projects#reset-the-default-compute-size). An ephemeral compute remains active for as long as you keep running queries against it. After 30 seconds of inactivity, the timeline is deleted and the endpoint is removed.
```

Comment:
Random thought: it's interesting that we mention them here, but we don't show them in the docs

Reply:
@ololobus We have it here: https://neon.tech/docs/guides/time-travel-assist#how-long-do-ephemeral-endpoints-remain-active
Maybe it's too buried.